src/main/java/gregtech/loaders/recipe/GTRecipeManager.java

```diff
@@ -20,6 +24,7 @@ public static void load() {
 
     public static void loadLatest() {
         DecompositionRecipeHandler.runRecipeGeneration();
+        MinecraftForge.EVENT_BUS.post(new MaterialInfoEvent());
```

Comment:
Shouldn't this be run before `DecompositionRecipeHandler`?

Reply:
I think so too it should as flags with which `DecompositionRecipeHandler` interacts may get modified.